src/main/java/org/broadinstitute/hellbender/utils/read/ReadUtils.java

```diff
@@ -888,17 +892,20 @@ public static SAMFileWriter createCommonSAMWriter(
             final File referenceFile,
             final SAMFileHeader header,
             final boolean preSorted,
-            boolean createIndex,
-            boolean createMD5)
+            final boolean createIndex,
+            final boolean createMD5)
     {
         Utils.nonNull(outputFile);
         Utils.nonNull(header);
 
-        if (createIndex && header.getSortOrder() != SAMFileHeader.SortOrder.coordinate) {
-            throw new UserException("Sort order must be coordinate to allow creation of an output index");
+        boolean createOutputBamIndex = createIndex;
```

Comment:
Why not just name the actual method parameter `createOutputBamIndex` instead of performing this assignment?